docs/howtos/redisai/getting-started/index-gettingstarted.mdx

```diff
@@ -0,0 +1,250 @@
+
+### Step 1. Installing RedisAI
+
+ ```bash
+  docker run \
+    -p 6379:6379 \
+    redislabs/redismod \
+    --loadmodule /usr/lib/redis/modules/redisai.so \
+      ONNX redisai_onnxruntime/redisai_onnxruntime.so
+ ```
+
+You will see that ONNX backend getting loaded as shown below in the results.
+
+ ```bash
+  1:C 09 Jun 2021 12:28:47.985 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
+  1:C 09 Jun 2021 12:28:47.985 # Redis version=6.0.1, bits=64, commit=00000000, modified=0, pid=1, just started
+  1:C 09 Jun 2021 12:28:47.985 # Configuration loaded
+  1:M 09 Jun 2021 12:28:47.987 * Running mode=standalone, port=6379.
+  1:M 09 Jun 2021 12:28:47.987 # Server initialized
+  1:M 09 Jun 2021 12:28:47.987 # WARNING you have Transparent Huge Pages (THP) support enabled in your kernel. This will create latency and memory usage issues with Redis. To fix this issue run the command 'echo never > /sys/kernel/mm/transparent_hugepage/enabled' as root, and add it to your /etc/rc.local in order to retain the setting after a reboot. Redis must be restarted after THP is disabled.
+  1:M 09 Jun 2021 12:28:47.989 * <ai> Redis version found by RedisAI: 6.0.1 - oss
+  1:M 09 Jun 2021 12:28:47.989 * <ai> RedisAI version 10003, git_sha=7f808a934dff121e188cb76fdfcc3eb1f9ec7cbf
+  1:M 09 Jun 2021 12:28:48.011 * <ai> ONNX backend loaded from /usr/lib/redis/modules/backends/redisai_onnxruntime/redisai_onnxruntime.so
+  1:M 09 Jun 2021 12:28:48.011 * Module 'ai' loaded from /usr/lib/redis/modules/redisai.so
+  1:M 09 Jun 2021 12:28:48.011 * Ready to accept connections
+ ```
+
+You can verify if RedisAI module is loaded or not by running the below CLI
+
+ ```bash
+  127.0.0.1:6379> info modules
+  # Modules
+  module:name=ai,ver=10003,api=1,filters=0,usedby=[],using=[],options=[]
+
+  # ai_git
+  ai_git_sha:7f808a934dff121e188cb76fdfcc3eb1f9ec7cbf
+
+  # ai_load_time_configs
+  ai_threads_per_queue:1
+  ai_inter_op_parallelism:0
+  ai_intra_op_parallelism:0
+ ```
+
+
+### Step 2. Setup Python Environment
+
+Ensure that Python3.8+ is installed.
+
+ ```bash
+  brew install python
+ ```
+
+### Step 3. Install PIP
+
+ ```bash
+  pip install --upgrade pip
+ ```
+
+### Step 4. Clone the repository
+
+ ```bash
+  git clone https://github.com/redis-developer/redisai-iris
+ ```
+
+### Step 5. Install the dependencies
+
+ ```bash
+  pip install -r requirements.txt
+ ```
+
+
+### Step 6. Build the ONNX Model
+
+RedisAI supports DL/ML identifiers and their respective backend libraries, including:
+
+- TF: The TensorFlow backend
+- TFLITE: The TensorFlow Lite backend
+- TORCH: The PyTorch backend
+- ONNX: ONNXRuntime backend
+
+A complete list of supported backends is in the [release notes for each version](https://docs.redis.com/latest/modules/redisai/release-notes/redisai-1.0-release-notes/).
+
+ ```bash
+  python3 build.py
```

Comment:
Note: if using a virtual env this can be `python` doesn't need to be `python3`.

Reply:
You'r right. Let me fix it.